docs/otoc/otoc_example.ipynb

```diff
@@ -0,0 +1,761 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {
+    "cellView": "form",
+    "id": "UaBOLd1ZNZwC"
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "No project_id provided and environment variable GOOGLE_CLOUD_PROJECT not set.\n",
+      "Using a noisy simulator.\n"
+     ]
+    }
+   ],
+   "source": [
+    "import os\n",
+    "\n",
+    "# The Google Cloud Project id to use.\n",
+    "project_id = '' #@param {type:\"string\"}\n",
+    "\n",
+    "\n",
+    "use_noisy_simulator = False\n",
+    "if project_id == '' and 'GOOGLE_CLOUD_PROJECT' not in os.environ:\n",
+    "    print(\"No project_id provided and environment variable \"\n",
+    "          \"GOOGLE_CLOUD_PROJECT not set.\")\n",
+    "    use_noisy_simulator = True\n",
+    "else: \n",
+    "    os.environ['GOOGLE_CLOUD_PROJECT'] = project_id\n",
+    "\n",
+    "    def authenticate_user():\n",
+    "        \"\"\"Runs the user through the Colab OAuth process.\n",
+    "\n",
+    "        Checks for Google Application Default Credentials and runs interactive login \n",
+    "        if the notebook is executed in Colab. In case the notebook is executed in Jupyter notebook\n",
+    "        or other IPython runtimes, no interactive login is provided, it is assumed that the \n",
+    "        `GOOGLE_APPLICATION_CREDENTIALS` env var is set or `gcloud auth application-default login`\n",
+    "        was executed already.\n",
+    "\n",
+    "        For more information on using Application Default Credentials see \n",
+    "        https://cloud.google.com/docs/authentication/production\n",
+    "        \"\"\"\n",
+    "        in_colab = False\n",
+    "        try:\n",
+    "            from IPython import get_ipython\n",
+    "            in_colab = 'google.colab' in str(get_ipython())\n",
+    "        except: \n",
+    "            # Notebook is not executed within IPython. Assuming external authentication.\n",
+    "            return \n",
+    "\n",
+    "        if in_colab: \n",
+    "            from google.colab import auth      \n",
+    "            print(\"Getting OAuth2 credentials.\")\n",
+    "            print(\"Press enter after entering the verification code.\")\n",
+    "            auth.authenticate_user(clear_output=False)\n",
+    "            print(\"Authentication complete.\")\n",
+    "        else: \n",
+    "            print(\"Notebook is not executed with Colab, assuming Application Default Credentials are setup.\") \n",
+    "\n",
+    "    authenticate_user()\n",
+    "    print(\"Successful authentication to Google Cloud.\")\n",
+    "\n",
+    "processor_id = \"\" #@param {type:\"string\"}\n",
+    "if use_noisy_simulator or processor_id == \"\":\n",
+    "    print(\"Using a noisy simulator.\")\n",
+    "    sampler = cg.PhasedFSimEngineSimulator.create_with_random_gaussian_sqrt_iswap(\n",
+    "        mean=cg.SQRT_ISWAP_PARAMETERS,\n",
```

Comment:
no such thing SQRT_ISWAP_PARAMETERS. Do you mean SQRT_ISWAP_INV_PARAMETERS

(non-optional)

Reply:
Fixed. (1st PR was before Cirq 0.11 update)